tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
-    "target": "es2021",
+    "target": "ES2022",
     "lib": [
-      "ES2021"
+      "ES2022"
     ],
     "module": "Node16",
-    "moduleResolution": "Node16",
+    "moduleResolution": "node16",
```

Comment:
i think we should maybe be switching this to at least `node18` now

Reply:
I believe there is only node16 (see, https://www.typescriptlang.org/tsconfig/#moduleResolution), however `module` can be `node18` from ts 5.8 https://www.typescriptlang.org/tsconfig/#node16node18nodenext, so I will use that.